### DISCUSSION TITLE
Add ReactJS Example

### DISCUSSION THREAD

**Discussion by virurathore** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-js#214] [General]
Please can you add some example which can use from ReactJS or nidesh code

**Comment by MixMasterMitch** [comment]
What is "nidesh code" that you are referring to?

**Comment by MixMasterMitch** [comment]
I have modified the title of this request. Please modify it if you believe my modification is not appropriate.

**Comment by lherman-cs** [comment]
This is not a a dedicated example, but we have a similar issue thread here, https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-js/issues/45#issuecomment-629308809. It contains a working React example.

**Comment by DustinTheGreat** [comment]
Jeeze is this still open?

**Comment by hassanctech** [comment]
The issue is open and we welcome contributions!

**Comment by DustinTheGreat** [comment]
I just implemented it in React, ill try to contribute it this weekend=)

**Comment by DustinTheGreat** [comment]
still a work in progress but heres something. https://github.com/DustinTheGreat/ServerlessSecurityCam

**Comment by zhex900** [comment]
How do you import or include this SDK in a react? 
I only can get it working via script tag ` <script src="https://unpkg.com/amazon-kinesis-video-streams-webrtc/dist/kvs-webrtc.min.js"></script>`

Is it possible to do import like a npm module?

**Comment by pdcarroll** [comment]
I created a React hooks wrapper for this library for anyone interested in trying it:

https://github.com/pdcarroll/react-kinesis-webrtc

**Comment by kunimasu** [comment]
https://github.com/pdcarroll/react-kinesis-webrtc/blob/master/src/hooks/useViewer.ts#L117

This is very helpful. Recent Safari needs to set recvonly when video only as viewer.
Current example in this repo is not set it. so in my case example is not called peerConnection.onTrack.
